authors/apps/authentication/views.py

```diff
@@ -29,13 +50,25 @@ def post(self, request):
         return Response(serializer.data, status=status.HTTP_201_CREATED)
 
 
-class LoginAPIView(APIView):
+class LoginAPIView(generics.GenericAPIView):
+    """
+    post:
+    login existing user.
+    """
     permission_classes = (AllowAny,)
     renderer_classes = (UserJSONRenderer,)
     serializer_class = LoginSerializer
 
+
     def post(self, request):
-        user = request.data.get('user', {})
+
+        email = request.data.get('email',None)
```

Comment:
@PAbishai these two assignments can be made a one-liner i.e `a,b = value[a], value[b]`

Reply:
same applies for Registration view.

Reply:
Got it. Let me amend